etc/scipipe/ap_verify.yaml

```diff
@@ -11,12 +11,14 @@ template:
       display_name: hits2015
       name: ap_verify_ci_hits2015
       github_repo: lsst/ap_verify_ci_hits2015
+      gen3_pipeline: ${AP_VERIFY_DIR}/pipelines/ApVerifyWithFakes.yaml
       git_ref: master
       clone_timelimit: 15
     cosmos_pdr2: &dataset_cosmos_pdr2
       display_name: cosmos_pdr2
       name: ap_verify_ci_cosmos_pdr2
       github_repo: lsst/ap_verify_ci_cosmos_pdr2
+      gen3_pipeline: ${AP_VERIFY_DIR}/pipelines/ApVerifyWithFakes.yaml
```

Comment:
I have a question for @ktlim. I've written this on the assumption that the string would be passed as-is to the command line, and expanded there before being passed to `run_ci_dataset.sh`. However, on Jenkins I get the following error on trying to load the pipeline:
```
FileNotFoundError: [Errno 2] No such file or directory: '/j/ws/scipipe/ap_verify/cosmos_pdr2-master^gen3^ap_verify-tickets_dm-29341/run/null'
```

Did I misunderstand how environment variables work in this context?

Reply:
Argh, never mind, I was pulling the wrong config field. 🤦

Reply:
Never mind never mind, I'm still getting the same error after #807. @ktlim, can I trouble you to see if anything is obviously wrong?

Reply:
Fixed. Sorry for the noise, all.